src/test/java/de/zalando/aruha/nakadi/validation/JSONSchemaValidationTest.java

```diff
@@ -164,58 +163,6 @@ public void requireMetadataOccurredAt() {
     }
 
     @Test
-    public void requireMetadataOccurredAtToBeFormattedAsDateTime() {
```

Comment:
This tests were all grouped in a more appropriate layer, testing the RFC validator. All test cases were preserved and new ones were added.